lib/compress/inference.js

```diff
@@ -426,7 +440,7 @@ export function is_nullish(node, compressor) {
         return any(this.body, compressor);
     });
     def_may_throw(AST_Call, function(compressor) {
-        if (this.optional && is_nullish(this.expression, compressor)) return false;
+        if (is_nullish(this.expression, compressor)) return false;
```

Comment:
```suggestion
        if (is_nullish(this, compressor)) return false;
```

Right? Because if the callee is `null` or `undefined`, it definitely throws.